recipes/repeatscout/build.sh

```diff
@@ -3,4 +3,4 @@ set -x -e
 mkdir -p ${PREFIX}/bin
 make
 
-cp RepeatScout build_lmer_table ${PREFIX}/bin
+cp RepeatScout build_lmer_table compare-out-to-gff.prl filter-stage-1.prl filter-stage-2.prl merge-lmer-tables.prl ${PREFIX}/bin
```

Comment:
Are these `.prl` files Perl scripts? Then you need a Perl dependency.

In any case, please add tests for them.

Reply:
I didn't want to bother you. No worries it can wait tomorrow sure.
I have checked you're right they are perl scripts. I will check if they need particular module and add tests tomorrow.